pkgpanda/__init__.py

```diff
@@ -43,13 +43,25 @@
 username_regex = "^dcos_[a-z0-9_]+$"
 
 
+def _is_root():
+    return os.geteuid() == 0
+
+
+def _set_sysctl(sysctl_value, sysctl_var):
+    try:
+        sysctl_setting = "{key}={value}".format(key=sysctl_var, value=sysctl_value)
+        check_call(["/sbin/sysctl", "-w", sysctl_setting])
```

Comment:
This is likely not in `/sbin` on some systems where we run. For instance, on my distro it is in `/usr/bin`